### ISSUE TITLE
Adaptations for other programming languages?

### ISSUE THREAD

**Issue by xiaochuanyu** [cosmicpython/book#336]
Is there any plans to adapt the content of this book to other programming languages?
For example, a Ruby version with Ruby examples & modified content.
I see that the licensing doesn't allow derivatives (Creative Commons CC-By-ND).
I guess this mean it's not possible?

**Comment by hjwp** [comment]
I don't think you could write a whole other book.  but if you wanted to write a series of blog posts called "following thru APwP in Ruby" which mostly consisted of code listings and discussions of the different ways of doing things in Ruby vs Python, then that would be a standalone work, not really a derivative work.  i'm not a lawyer and i can't speak for OR but i don't think they would mind.  and i personally would be glad of it, because presumably people would want to go and read the source chapter for each blog post...
